frontend/www/js/omegaup/components/course/Details.vue

```diff
@@ -0,0 +1,260 @@
+                <a
+                  v-bind:href="
+                    `/course/${course.alias}/assignment/${exam.alias}/scoreboard/${exam.scoreboard_url}`
+                  "
+                >
+                  <font-awesome-icon v-bind:icon="['fas', 'link']" />
+                  {{ T.wordsPublic }}
+                </a>
+                <a
+                  v-bind:href="
+                    `/course/${course.alias}/assignment/${exam.alias}/scoreboard/${exam.scoreboard_url_admin}`
+                  "
+                >
+                  <font-awesome-icon v-bind:icon="['fas', 'link']" />
+                  {{ T.wordsAdmin }}
+                </a>
+                <a
+                  v-bind:href="
+                    `/course/${course.alias}/assignment/${exam.alias}/admin/#runs`
+                  "
+                >
+                  <font-awesome-icon v-bind:icon="['fas', 'tachometer-alt']" />
+                  {{ T.wordsRuns }}
+                </a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <div class="card-footer">
+        <a
+          class="btn btn-primary float-right"
+          v-bind:href="`/course/${course.alias}/edit/#assignments/new/test/`"
+          >{{ T.wordsNewExam }}</a
+        >
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import T from '../../lang';
+import * as ui from '../../ui';
+import * as time from '../../time';
+import * as markdown from '../../markdown';
+import { types } from '../../api_types';
+
+import { VueMathjax } from 'vue-mathjax';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome';
+import {
+  faEdit,
+  faLink,
+  faTachometerAlt,
+} from '@fortawesome/free-solid-svg-icons';
+library.add(faEdit, faLink, faTachometerAlt);
+
+@Component({
+  components: {
+    FontAwesomeIcon,
+    'vue-mathjax': VueMathjax,
+  },
+})
+export default class CourseDetails extends Vue {
+  @Prop() course!: omegaup.Course;
+  @Prop() progress!: types.AssignmentProgress[];
+
+  T = T;
+  ui = ui;
+  markdownConverter = markdown.markdownConverter();
+
+  get filteredHomeworks(): omegaup.Assignment[] {
+    return this.course.assignments.filter(
+      assignment => assignment.assignment_type === 'homework',
+    );
+  }
+
+  get filteredExams(): omegaup.Assignment[] {
+    return this.course.assignments.filter(
+      assignment => assignment.assignment_type === 'test',
+    );
+  }
+
+  getAssignmentProgress(progress: types.Progress): string {
+    const percent = (progress.score / progress.max_score) * 100;
+    const percentText = progress.max_score === 0 ? '--:--' : percent.toFixed(2);
+    return progress.max_score === 0 ? percentText : `${percentText}%`;
+  }
+
+  getFormattedTime(timestamp: number): string {
```

Comment:
según `api_types`, todos los timestamps ya son fechas!

```suggestion
  getFormattedTime(timestamp: Date): string {
```